### DISCUSSION TITLE
[SOLVED] Re-add .npmignore file

### DISCUSSION THREAD

**Discussion by pioardi** [poolifier/poolifier#238] [Q&A]
I would re-add the .npmignore file since that npm quality score checks for it.
Do you have any concern doing that?

@jerome-benoit @Shinigami92

**Comment by Shinigami92** [comment]
Could you link a docu for that? I mean, why should npm check such a file for quality score?! Reads strange to me.

**Reply by pioardi** [reply]
Nevermind it also checks for file includes.
If you are curious anyway this is an intereseting article https://itnext.io/increasing-an-npm-packages-search-score-fb557f859300
